src/main/java/org/kairosdb/core/properties/PropertiesValidator.java

```diff
@@ -0,0 +1,54 @@
+package org.kairosdb.core.properties;
+
+import com.google.inject.Inject;
+import com.google.inject.name.Named;
+import org.kairosdb.core.exception.KairosDBException;
+
+/**
+ * Ensure that nothing in the kairosdb.properties file would result in
```

Comment:
Is the plan to validate all configuration in one file? Or will there be different validators for different logical validations?

Reply:
For right now, all in one file, with each check being its own function. Is it great? No. Is it functional and an improvement over the status quo? I believe yes.